services/clinical/index.js

```diff
@@ -73,6 +73,17 @@ const getClinicalSubmissionTypesList = async () => {
   return response;
 };
 
+const getClinicalSubmissionSchemaVersion = async () => {
```

Comment:
let's add some jsdoc to new utilities :) https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html#supported-jsdoc

vscode has some autocomplete built-in so that should help ;)

Reply:
in the end I added for this one only because I don't remember what the return type is for the other ones :stuck_out_tongue: